artifacts/attributes.adoc

```diff
@@ -2,6 +2,8 @@
 :product: Red Hat Developer Hub
 :product-short: Developer Hub
 :product-version: 1.1
```

Comment:
should this move to 1.2 in main?

Reply:
Yes for the main branch, it should be 1.2